folium/folium.py

```diff
@@ -284,10 +288,17 @@ def __init__(
         self.crs = crs
         self.control_scale = control_scale
 
+        # Zoom control position specified ?
+        if isinstance(zoom_control, str):
+            self.zoom_control_position = True
+            self.zoom_control = zoom_control
```

Comment:
maybe useful to add a check here for valid values? Now if a users makes an error, they just get a grey screen. For example something like:

```suggestion
            self.zoom_control = zoom_control
            if zoom_control not in {'topleft', 'topright', 'bottomleft', 'bottomright'}:
                raise ValueError("Incorrect value for `zoom_control`, choose from 'topleft', 'topright', 'bottomleft' or 'bottomright'.")
```

Reply:
I had considered introducing a check on valid position values but looking at various plugins I could not find such a check so I thought it was considered as unnecessary ...

Reply:
I understand, this code base grew quite organically over time, so it's not always consistent. Thanks for putting it in!